vector/src/test/java/im/vector/app/features/command/CommandParserTest.kt

```diff
@@ -71,12 +71,7 @@ class CommandParserTest {
 
     private fun test(message: String, expectedResult: ParsedCommand) {
         val commandParser = CommandParser()
-<<<<<<< HEAD
-        val result = commandParser.parseSlashCommand(message, false)
-        result shouldBeIn arrayOf(expectedResult, ParsedCommand.ErrorNotATchapCommand)
-=======
         val result = commandParser.parseSlashCommand(message, null, false)
-        result shouldBeEqualTo expectedResult
->>>>>>> v1.5.7
+        result shouldBeIn arrayOf(expectedResult, ParsedCommand.ErrorNotATchapCommand)
```

Comment:
We should have a tchap comment.

Reply:
https://github.com/tchapgouv/tchap-android/pull/806/commits/d2c0f1114a43755a1e02a20cfa719c7dda7e235b